x-pack/plugins/observability/public/pages/rule_details/index.tsx

```diff
@@ -123,11 +125,15 @@ export function RuleDetailsPage() {
   const tabsRef = useRef<HTMLDivElement>(null);
 
   const onAlertSummaryWidgetClick = async (status: AlertStatus = ALERT_STATUS_ALL) => {
+    const timeRange = getAlertSummaryWidgetTimeRange();
+    setAlertSummaryWidgetTimeRange(timeRange);
```

Comment:
💭 When we click Alert Summary Widget, the time range will be updated for both the widget and table to make sure they are up to date and matched with each other.